articles/ctod.dd

```diff
@@ -646,9 +646,13 @@ ulong[string] map = ["hello"    : 1616515,
                      "goodbye"  : 42,
                      "maybe"    : ulong.max
                     ];
-string s = "goodbye";
-
-writeln("number is ", map[s]);
+unittest
```

Comment:
I don't know that unittest blocks are compiled using the RUNNABLE_EXAMPLE_ macros.

Reply:
It seems the Compile button does compile them, but DAutoTest does not.

Reply:
`tools/dspec_tester.d` doesn't look at `articles/`. I'm looking at fixing that but running into other issues too. Hopefully I can fix it.